keras_tuner/tuners/bayesian.py

```diff
@@ -122,37 +122,34 @@ class BayesianOptimizationOracle(oracle_module.Oracle):
 
-    Arguments:
-        objective: String or `keras_tuner.Objective`. If a string,
-          the direction of the optimization (min or max) will be
-          inferred.
-        max_trials: Int. Total number of trials
-            (model configurations) to test at most.
-            Note that the oracle may interrupt the search
-            before `max_trial` models have been tested if the search space has been
-            exhausted.
-        num_initial_points: (Optional) Int. The number of randomly generated samples
-            as initial training data for Bayesian optimization. If not specified,
-            a value of 3 times the dimensionality of the hyperparameter space is
-            used.
-        alpha: Float. Value added to the diagonal of the kernel matrix
-            during fitting. It represents the expected amount of noise
-            in the observed performances in Bayesian optimization.
-        beta: Float. The balancing factor of exploration and exploitation.
-            The larger it is, the more explorative it is.
-        seed: Int. Random seed.
-        hyperparameters: HyperParameters class instance.
-            Can be used to override (or register in advance)
-            hyperparameters in the search space.
-        tune_new_entries: Whether hyperparameter entries
-            that are requested by the hypermodel
-            but that were not specified in `hyperparameters`
-            should be added to the search space, or not.
-            If not, then the default value for these parameters
-            will be used.
-        allow_new_entries: Whether the hypermodel is allowed
-            to request hyperparameter entries not listed in
-            `hyperparameters`.
+    Args:
+        objective: A string or `keras_tuner.Objective` instance. If a string,
+            the direction of the optimization (min or max) will be inferred.
+        max_trials: Integer, the total number of trials (model configurations)
+            to test at most. Note that the oracle may interrupt the search
+            before `max_trial` models have been tested if the search space has
+            been exhausted.
+        num_initial_points: Optional number of randomly generated samples as
+            initial training data for Bayesian optimization. If left
+            unspecified, a value of 3 times the dimensionality of the
+            hyperparameter space is used.
+        alpha: Float, the value added to the diagonal of the kernel matrix
+            during fitting. It represents the expected amount of noise in the
+            observed performances in Bayesian optimization. Defaults to 1e-4.
+        beta: Float, the balancing factor of exploration and exploitation. The
+            larger it is, the more explorative it is. Defaults to 2.6.
+        seed: Optional integer, the random seed.
+        hyperparameters: Optional HyperParameters instance. Can be used to
```

Comment:
Use backticks around HyperParameters, since we're talking about an instance of the class